examples/variational_autoencoder.py

```diff
@@ -39,7 +41,7 @@ def sampling(args):
 
 
 def vae_loss(x, x_decoded_mean):
-    xent_loss = original_dim * objectives.binary_crossentropy(x, x_decoded_mean)
+    xent_loss = objectives.binary_crossentropy(x, x_decoded_mean)
```

Comment:
Thanks for the review @EderSantana! Do you know any reference using conv+vae? I found [this one](http://people.ee.duke.edu/~lcarin/Yunchen_nips_2016.pdf). The encoder-decoder part seems similar to the example.